internal/generator/vector/normalize.go

```diff
@@ -12,13 +12,13 @@ import (
 const (
 	FixLogLevel = `
 level = "unknown"
-if match(.message,r'(Warning|WARN|W[0-9]+|level=warn|Value:warn|"level":"warn")'){
+if match!(.message,r'(Warning|WARN|W[0-9]+|level=warn|Value:warn|"level":"warn")'){
```

Comment:
How does this change relate to that described in the bug?

Reply:
not related directly. 
Since i was suspecting that journal source is not working in vector version i was using (the metrics also showed 0),  I wanted to check this bug on latest version of vector. 
The latest vector gave an error for these lines.